MovieAppPetProject/MovieAppPetProject/GetMovieDetailsByID/GetMovieDetails-ViewModel.swift

```diff
@@ -7,16 +7,20 @@
 
 import Foundation
```

Comment:
why do your files have a dash in then for the name?
should just be MovieDetailsViewModel.swift
remove Get as well

Reply:
understood